src/py/mains/ovlp_stats.py

```diff
@@ -44,14 +42,13 @@ def filter_stats(readlines, min_len):
             if idt < 90:
                 continue
 
-            if not contained:
-                if l[-1] in ("contains", "overlap"):
-                    ave_idt += idt * overlap_len
-                    all_over_len += overlap_len
-                if q_s == 0:
-                    overlap_data["5p"] += 1
-                elif q_e == q_l:
-                    overlap_data["3p"] += 1
+            if l[-1] in ("contains", "overlap"):
+                ave_idt += idt * overlap_len
+                all_over_len += overlap_len
+            if q_s == 0:
+                overlap_data["5p"] += 1
+            if q_e == q_l:
```

Comment:
Oh, wait. Now I see! This was the same fix made to ovlp_filter.py, but applied to ovlp_stats.py
* https://github.com/PacificBiosciences/FALCON/pull/235/files#diff-69cae0e09e73dce9913b24c5dcf32292R56